iothub/device/src/Exceptions/IotHubClientException.cs

```diff
@@ -185,7 +185,7 @@ internal IotHubClientException(bool isTransient, IotHubStatusCode statusCode) :
         public string TrackingId { get; set; }
 
         /// <summary>
-        /// The HTTP status statusCode returned by the IoT hub service.
+        /// The HTTP status code returned by the IoT hub service.
         /// </summary>
         public IotHubStatusCode StatusCode { get; internal set; }
```

Comment:
ErrorCode to match service SDK?

Reply:
No, this is property in `IotHubClientException` thrown by the **hub device**. The changes here is to fix the doc comments only.

Reply:
And the code will be simply 200, 400, 500? Or the 6-digit kind?

Reply:
Discussed with Bryce offline and we need to gather the code from DPS which are 6-digits.

Reply:
> And the code will be simply 200, 400, 500? Or the 6-digit kind?

it's more like 6-digit kind. FYI, I am revamping this `IotHubClientException` to be more aligned with the `IotHubServiceException` by using 2 properties `StatusCode`(3-digit) and `ErrorCode`(6-digit).

Reply:
"revert" the changes related to "hub client exceptions" in this PR as I am having another PR for `IotHubClientException` in particular.